index.html

```diff
@@ -17,7 +17,7 @@
         navigator.gpu.requestAdapter().then(v => { if (v != null) window.location = location.href + "/webgpu" + location.search }, _ => {})
     }
 
-    //document.addEventListener('contextmenu', event => event.preventDefault());
+    document.addEventListener('contextmenu', event => event.preventDefault());
     //window.addEventListener('scroll', event => event.preventDefault());
```

Comment:
I'm fairly sure I added this for a reason, I think allowing scroll breaks stuff on mobile or something